grr/server/grr_response_server/gui/ui/components/client/client.ng.html

```diff
@@ -6,6 +6,7 @@ <h1 class="mat-h1">{{client.knowledgeBase.fqdn}}</h1>
       </div>
       <div class="chips">
         <mat-chip-list>
+          <status-chip [lastSeen]="client.lastSeenAt"></status-chip>
```

Comment:
We also want to use this component outside of <Client />. For unambiguity, please rename the component to  `online-chip`, `client-online-chip`, or something of your choice.

Reply:
Yes, no problem